Meadowlark-js/.env.example

```diff
@@ -27,4 +27,7 @@ POSTGRES_PORT=5432
 
 # Do not override MongoDB settings here, because the integration tests use an "in memory"
 # version of MongoDB that is configured automatically. Any settings applied here will
-# cause the connection to fail.
\ No newline at end of file
+# cause the connection to fail.
```

Comment:
The above message should be changed, since we're clearly overriding a MongoDB setting.